examples/mqtt/mqtt-shapes/README.md

```diff
@@ -180,6 +180,12 @@ The following additional external dependencies are required to run the demo:
 
 - [tmux](https://github.com/tmux/tmux) (optional)
```

Comment:
We should probably rephrase the "Dependencies" section a bit, to something like:

"The demo relies on some external dependencies: ..."

That way is less inconsistent that tmux is marked optional in a list that says "required".